configuration/dto.go

```diff
@@ -56,7 +56,6 @@ type RealmAdminConfiguration struct {
 	NeedVerifiedContact                            *bool           `json:"need_verified_contact,omitempty"`
 	ConsentRequiredSocial                          *bool           `json:"consent_required,omitempty"`
 	ConsentRequiredCorporate                       *bool           `json:"consent_required_corp,omitempty"`
-	ShowGlnEditing                                 *bool           `json:"show_gln_editing,omitempty"`
```

Comment:
Take care: you'll have to ensure you can unmarshal JSON with the show_gln_editing field present in the structure.